src/query/storages/fuse/Cargo.toml

```diff
@@ -32,6 +32,7 @@ common-sql = { path = "../../sql" }
 common-storage = { path = "../../../common/storage" }
 jsonb = { workspace = true }
 
+common-storages-stage = { path = "../stage" }
```

Comment:
maybe we can put it in `databend-query::table_functions` to avoid adding this dep?
since it not rely on internal fn of `common-storages-fuse`.

so does `infer_schema`?

cc @BohuTANG

Reply:
![image](https://user-images.githubusercontent.com/5782159/226249019-3d578ede-d860-4d6b-abbe-12eff6df2605.png)